src/plugins/minecraft.rs

```diff
@@ -4,40 +4,50 @@ use crate::prelude::*;
 
 pub struct MinecraftPlugin;
 
+impl MinecraftPlugin {
+    async fn handle_mc_players(&self, ctx: &Context, arg: &str) -> Result<()> {
+        let parts: Vec<&str> = arg.splitn(2, ':').collect();
+        let address = parts
+            .get(0)
+            .map(|s| (*s).to_string())
```

Comment:
`cargo clippy`?

Reply:
Yeah, that's what told me to do this. There's an optimized version of .to_string for str, rather than &str.